<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>349</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <italic>true</italic>
    <bold>true</bold>
    <strikeout>false</strikeout>
    <stylestrategy>PreferAntialias</stylestrategy>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Diccionario del Ingles al Espa√±ol</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtWord">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="accessibleName">
     <string extracomment="asfdaf"/>
    </property>
    <property name="inputMask">
     <string extracomment="ddwefsafaf"/>
    </property>
    <property name="placeholderText">
     <string comment="kjndfd" extracomment="palabra a buscar"/>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>20</width>
      <height>201</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>16</width>
      <height>201</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelWithList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>With List</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWithHash">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>With Hash</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWithSet">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>With Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeList">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeHash">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeSet">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTimeList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTimeHash">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTimeSet">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTraducir">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Traducir</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessageList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessageHash">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessageSet">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtWithList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtWithHash">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtWithSet">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionload_in_List"/>
    <addaction name="actionload_in_Hash"/>
    <addaction name="actionload_in_Set"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionload_in_List"/>
   <addaction name="actionload_in_Hash"/>
   <addaction name="actionload_in_Set"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionload_in_List">
   <property name="icon">
    <iconset resource="resourcet.qrc">
     <normaloff>:/new/prefix1/listas.jpeg</normaloff>:/new/prefix1/listas.jpeg</iconset>
   </property>
   <property name="text">
    <string>load in &amp;List</string>
   </property>
  </action>
  <action name="actionload_in_Hash">
   <property name="icon">
    <iconset resource="resourcet.qrc">
     <normaloff>:/new/prefix1/Tabla_hash.png</normaloff>:/new/prefix1/Tabla_hash.png</iconset>
   </property>
   <property name="text">
    <string>load in &amp;Hash</string>
   </property>
  </action>
  <action name="actionload_in_Set">
   <property name="icon">
    <iconset resource="resourcet.qrc">
     <normaloff>:/new/prefix1/set.png</normaloff>:/new/prefix1/set.png</iconset>
   </property>
   <property name="text">
    <string>load in &amp;Set</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resourcet.qrc">
     <normaloff>:/new/prefix1/close.jpeg</normaloff>:/new/prefix1/close.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resourcet.qrc">
     <normaloff>:/new/prefix1/about.jpeg</normaloff>:/new/prefix1/about.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourcet.qrc"/>
 </resources>
 <connections/>
</ui>
